### PR TITLE
CMR-8370 Fixing the UMM-C migration to and from 1.17.0.  If orbit par…

### PR DESCRIPTION
**Pull request by eereiter**
…ameters doesn't exist then don't change it.

### CODE REVIEW

File: umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj [hunk 1]
```diff
@@ -123,17 +123,19 @@
...
-        collection (-> collection
-                       (update-in [:SpatialExtent :OrbitParameters] dissoc :Period)
-                       (update-in [:SpatialExtent :OrbitParameters] assoc :SwathWidthUnit "Kilometer"
-                                                                          :OrbitPeriodUnit "Decimal Minute"
-                                                                          :InclinationAngleUnit "Degree"
-                                                                          :OrbitPeriod orbit-period))]
-    (if StartCircularLatitude
-      (assoc-in collection [:SpatialExtent :OrbitParameters :StartCircularLatitudeUnit] "Degree")
-      collection)))
+  (if (get-in collection [:SpatialExtent :OrbitParameters])
```
Comment (daniel-zamora) [inline umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj:126]:
```
(if-let [orbit-period (get-in collection [:SpatialExtent :OrbitParameters :Period])]
  (let [...
```
Works better here

File: umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj [hunk 2]
```diff
@@ -166,12 +168,14 @@
   convert SwathWidth to the value in assumed unit; If SwathWidth doesn't exist,
   convert largest Footprint to SwathWidth."
   [collection]
-  (let [swath-width (get-swath-width collection)
-        period (get-in collection [:SpatialExtent :OrbitParameters :OrbitPeriod])]
-    (-> collection
-        (update-in [:SpatialExtent :OrbitParameters] dissoc :Footprints :OrbitPeriod :SwathWidthUnit :OrbitPeriodUnit
-                :InclinationAngleUnit :StartCircularLatitudeUnit)
-        (update-in [:SpatialExtent :OrbitParameters] assoc :SwathWidth swath-width :Period period))))
+  (if (get-in collection [:SpatialExtent :OrbitParameters])
```
Comment (daniel-zamora) [inline umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj:171]: See other comment

**Review by daniel-zamora — APPROVED** [review]
Few small comments that need addressing, but otherwise approved.

File: umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj (hunk 1 above)
Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj]: I have changed the file.

File: umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj (hunk 2 above)
Comment (eereiter) [reply umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj]: I have updated the code.

**Review by TylerHeald1 — APPROVED** [review]

**Pull request merged**